src/scss/components/_inputs.scss

```diff
@@ -84,7 +84,9 @@ $input-select-margin-right: 1;
 
 .usa-checkbox__input:checked + .usa-checkbox__label::before,
 .usa-checkbox__input:checked:disabled + .usa-checkbox__label::before {
+  background-color: color('disabled');
   background-size: units(1) auto;
+  box-shadow: inset 0 0 0 units($border-width) color('disabled');
```

Comment:
Also, these are currently only applying to checkboxes. Radio fields look significantly lighter in the current version of the branch. Is that intended?

![image](https://user-images.githubusercontent.com/1779930/168613043-14907593-14b5-4cad-841f-8c665c202717.png)